yarn-project/circuits.js/src/structs/content_commitment.ts

```diff
@@ -31,9 +31,9 @@ export class ContentCommitment {
   toFields(): Fr[] {
     const serialized = [
       this.txTreeHeight,
-      ...toTruncField(this.txsEffectsHash),
-      ...toTruncField(this.inHash),
-      ...toTruncField(this.outHash),
+      toTruncField(this.txsEffectsHash),
+      toTruncField(this.inHash),
+      toTruncField(this.outHash),
```

Comment:
This and other methods (like `fromFields`) are simply for formatting the data given to this class. We should expect the input values to be truncated (or processed in any way needed) already. Maybe can check them in the constructor if we want to avoid any confusing error downstream when given a random buffer that's not a truncated field.